apps/acme-assist/pom.xml

```diff
@@ -32,17 +32,27 @@
 		<dependency>
 			<groupId>com.azure</groupId>
 			<artifactId>azure-ai-openai</artifactId>
-			<version>1.0.0-beta.3</version>
+			<version>1.0.0-beta.7</version>
 		</dependency>
 		<dependency>
-			<groupId>org.springframework.experimental.ai</groupId>
+			<groupId>org.springframework.ai</groupId>
```

Comment:
nit: seems like they have a bom for spring ai https://docs.spring.io/spring-ai/reference/getting-started.html